index.html

```diff
@@ -2037,33 +2051,38 @@ <h3>Verify Proof</h3>
       <section>
         <h3>Verify Proof Sets and Chains</h3>
         <p>
-In a proof set or proof chain, a <a>secured data document</a> has a
-<code>proof</code> attribute which contains a list of proofs
+In the case of a proof set or proof chain a <a>secured data document</a> has a 
```

Comment:
this feels like a repeat suggestion...
```suggestion
In a proof set or proof chain, a <a>secured data document</a> has a
```